features/manage-position/Withdraw.tsx

```diff
@@ -130,7 +128,7 @@ const Deposit = () => {
       : startingCR;
   const resultingCRBelowGCR = resultingCR && gcr ? resultingCR < gcr : null;
   const fastWithdrawableCollateral =
-    collateral && tokens && gcr ? (collateral - gcr * tokens).toFixed(2) : null;
+    collateral && tokens && gcr ? (gcr * tokens - collateral).toFixed(2) : null;
```

Comment:
Is this correct?

Reply:
previously the `fastWithdrawableCollateral` was showing me a negative value. see screenshot
![image](https://user-images.githubusercontent.com/12886084/88366236-e9348100-cd88-11ea-8573-5fc1cd7924d2.png)

so I swapped it around.

Thinking about this again now this is because my position is below the GCR! that's why it's negative. there is 0 that I can fast withdrawal. I'll change this to say if < 0 then it should = 0 so there is no negative case.

Reply:
Yes I agree. This should be changed to:

```
  const fastWithdrawableCollateral =
    collateral !== null && tokens !== null && gcr !== null
      ? (collateral < gcr * tokens ? 0 : (collateral - gcr * tokens).toFixed(2))
      : null;
```